CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.4.0
+
+### Known Issues
+
+### Features Added
+- Multi-key signing: Ethsigner is initialised with a directory containing a number of TOML metadata files, which describe the keys available for signing. Upon reception of a Transaction, Ethsigner loads the appropriate metadata file, and signs the Transaction with the associated key.
```

Comment:
Should it be _corresponding_ metadata file, given you need to explicitly specify the mapping?  Imo _appropriate_ sounds like Ethsigner can choose what to do with a given input.